### ISSUE TITLE
How to get the validation set

### ISSUE THREAD

**Issue by pangtouyuqqq** [york135/CTC_CE_for_AST#1]
Hi, thanks for open source your wonderful work~!
I am trying to reproduce your work, but I am confused at how to get the validation set.

In your paper, you said 
_In a preliminary experiment, we split the dataset to create a validation set, and found that using the training set or validation set for hyper-parameter search leads to similar results. Therefore, for simplicity, instead of separating the validation set from the training set, we simply tune the hyper-parameters on the training set._

So I just have to set the `val_dataset_dir` the same as `training_dataset_dir` ?
https://github.com/york135/CTC_CE_for_AST/blob/main/configs/train_config.yaml#L9

**Comment by york135** [comment]
Hi,
Yes, I think that's OK. The validation part is just a legacy, and does not affect the training process.

**Comment by pangtouyuqqq** [comment]
Emmmmm, In my opinion, neither using trainingset or testset as the validation set is the conventional practice in ML.
[Why Can't I use Test set as Validation Set](https://stats.stackexchange.com/questions/237790/why-cant-i-use-test-set-as-validation-set)

**Comment by york135** [comment]
Well, then you can simply delete the validation code......
I DID NOT USE THE TEST SET FOR VALIDATION. As I said, that's just a legacy and I forgot to remove it. I IGNORED the so-called validation process during my experiment, and the test set is just a placeholder. Apparently I should reorganize the code if I have time to avoid misunderstandings.

**Comment by pangtouyuqqq** [comment]
Got it~! My mistake, I misunderstand your response and your code. Sorry about it~ (。・＿・。)ﾉ
